runtime/jcl/common/jclcinit.c

```diff
@@ -773,6 +773,9 @@ initializeRequiredClasses(J9VMThread *vmThread, char* dllName)
 		return 1;
 	}
 	vmThread->outOfMemoryError = oom;
+#if JAVA_SPEC_VERSION >= 15
+	vm->nullPointerExceptionClass = vmFuncs->internalFindKnownClass(vmThread, J9VMCONSTANTPOOL_JAVALANGNULLPOINTEREXCEPTION, J9_FINDKNOWNCLASS_FLAG_NON_FATAL);
```

Comment:
Rather than doing this, you could just preload NPE using the existing mechanism and fetch it via J9VMJAVALANGNULLPOINTEREXCEPITON_OR_NULL.

Reply:
Sure, will switch to `J9VMJAVALANGNULLPOINTEREXCEPTION_OR_NULL(vm)` assuming NPE is one of the known classes initialized within `initializeKnownClasses`.

Reply:
For java15, add NPE to the list of required classes that already exists.

Reply:
Updated the PR, it works with following simple testcases:
```
new NullPointerException().printStackTrace();
java.lang.NullPointerException
	at a.se.next.TestNPEMsg.testAll(TestNPEMsg.java:8)
	at a.se.next.TestNPEMsg.main(TestNPEMsg.java:13)

new NullPointerException("JDK15 NPE").printStackTrace();
java.lang.NullPointerException: JDK15 NPE
	at a.se.next.TestNPEMsg.testAll(TestNPEMsg.java:9)
	at a.se.next.TestNPEMsg.main(TestNPEMsg.java:13)
```
Please have another look.